config/crd/bases/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -441,6 +441,11 @@ spec:
                         type: string
                     type: object
                 type: object
+              enableDataMover:
```

Comment:
run `make bundle` to propagate CRD changes to bundle/

Reply:
Done updated the PR